code/game/objects/items/toys.dm

```diff
@@ -1091,4 +1091,33 @@ obj/item/toy/cards/deck/syndicate
 	desc = "An adorable stuffed toy that resembles a space carp."
 	icon = 'icons/obj/toy.dmi'
 	icon_state = "carpplushie"
-	w_class = 2.0
\ No newline at end of file
+	w_class = 2.0
+
+/obj/machinery/toy_supermatter
+	name = "supermatter shard"
+	desc = "A fake variant of the infamous supermatter crystal."
+	icon = 'icons/obj/supermatter.dmi'
+	icon_state = "darkmatter_shard"
+	density = 1
+	anchored = 0
+	luminosity = 4
+	var/safe_alert = "Crystalline hyperstructure returning to safe operating levels."
+	var/warning_alert = "Danger! Crystal hyperstructure instability!"
+	var/emergency_alert = "CRYSTAL DELAMINATION IMMINENT."
+	var/obj/item/device/radio/radio
+	var/stage = 1

```

Comment:
should start at stage 2, because otherwise the first use will 'return' it to safe operating levels
this comment might end up a line higher than it should be because github's a dong